experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/list.html

```diff
@@ -3,6 +3,11 @@
 {% load static %}
 {% load nimbus_extras %}
 
+{# djlint:off #}
+{% block page_id %}PageHome-page{% endblock page_id %}
+
+
```

Comment:
djlint seems to have a bug where it'll inject multiple newlines on multiple runs.  I've seen this a couple times now.  It seems to stop at 2.